Source/engine.cpp

```diff
@@ -228,7 +228,8 @@ int32_t GenerateRnd(int32_t v)
 size_t GetFileSize(const char *pszName)
 {
 	HANDLE file;
-	SFileOpenFile(pszName, &file);
+	if (!SFileOpenFile(pszName, &file))

```

Comment:
Are we now passing filenames for files that do not exist?

Reply:
No this is only needed for the unittests.
They now also try to load the animations, cause it's now also part of `NewPlrAnim`.
And without this conditions they would crash later.

If you want I could change the condition to also check `gbQuietMode`.

Reply:
That might be a good idea in the rare cases where people have a corrupt diabdat.mpq

Reply:
I added a app_fatal call that would hopefully help to find this cases easier.